TASVideos/Pages/AwardsEditor/Assign.cshtml.cs

```diff
@@ -49,6 +50,21 @@ public async Task<IActionResult> OnPost()
 			ModelState.AddModelError("", "Cannot assign both a user and a publication to an award.");
 		}
 
+		var type = _awards.AwardCategories()
+			.Where(c => c.ShortName == AwardToAssign.Award)
+			.Single()
```

Comment:
This was questioned on discord but I dunno how do do it better. Currently it crashes because there's more than one award with the same name, because it allowed assiging the same award to user and movie if it was done separately. The protection was only against doing it in one go.

Reply:
.Single() wil throw an exception if there is 0 ore more than 1.  That will result in a 500 error.  Maybe you want to do a .ToList() and check how many there are and return a ModelState error?

Reply:
Sorry I misremembered. This one should be fine, because we only have the award category here because it was selected from the dropdown
https://github.com/TASVideos/tasvideos/blob/78bf0969adbefacccfb06fcd1bfefaf9a4318406/TASVideos/Pages/AwardsEditor/Assign.cshtml#L15
which gets populated from the same source
https://github.com/TASVideos/tasvideos/blob/78bf0969adbefacccfb06fcd1bfefaf9a4318406/TASVideos/Pages/AwardsEditor/Assign.cshtml.cs#L107
and we're simply checking what type that category has.

Reply:
So I don't have anything to add here, because `AwardToAssign.Award` is guaranteed to match at least 1 `ShortName`, since it's taken from that list in `Initialize()`. And it can't have several matches unless someone hacks the DB, which seems to have happened in 2021 before we had this UI, and 2 duplicate awards were created, which were removed yesterday.